src/styles/blocks/card.css

```diff
@@ -0,0 +1,96 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__photo {
+  display: flex;
+  object-fit: contain;
+  width: 100%;
+  height: 174px;
```

Comment:
Looks strange, this picture have it's own sizes in mockup, and indents on the bottom and on both left and right sides, so calculating of height to this image - is stranger, then just margin-bottom 🙂